### ISSUE TITLE
Support multiple accounts per social network?

### ISSUE THREAD

**Issue by webknjaz** [alexandru/social-publish#54]
Hi, I'm looking for a solution to use for a few accounts of open source projects. For example, I've recently registered a pair of accounts for @aio-libs (the org) and aiohttp (a project) on both Bluesky and Mastodon.
I can imagine wanting to make more accounts for other projects in the future (I happen to be involved in many).
I also want to share the ability to author the posts into all of these with co-maintainers, and maybe even integrate into the release processes via CI.

Looking into the README, I see that it expects a single set of credentials for each of the social networks to be supplied through the env vars.

So my question is whether you're planning to extend the functionality. If not — would it make sense to attempt running multiple instances of the service? Any other ideas?

I'm in research mode, and I'm looking for solutions, but since I landed here, I figured I'd ask.

**Comment by alexandru** [comment]
Unsure if you still want this or not (two years later :)), but ...

The project supports multiple user accounts now (new users can be created via the CLI), so that's a partial fix (would work great with Firefox's containers for instance).

**Comment by webknjaz** [comment]
Yeah, I still haven't really implemented what I wanted. I've been looking at https://github.com/woodruffw/autopost too. Thanks for the update!

**Comment by webknjaz** [comment]
Could you link to some explanation of how the multiuser mode works or where to start. Skimming through the README or that commit doesn't fully show that..
